README.md

```diff
@@ -17,13 +17,14 @@ A curated list of awesome dhall-lang binding, libraries and anything related to
 - [Miscellaneous](#miscellaneous)
 
 ## Binding
-- [dhall-haskell](https://github.com/dhall-lang/dhall-haskell)
-- [dhall-eta](https://github.com/eta-lang/dhall-eta) - A eta wrapper of dhall-haskell with the goal of provide a friendly java api over dhall.
-- [dhall-clj](https://github.com/f-f/dhall-clj) - Dhall to and from Clojure.
-- [dhall-purescript](https://github.com/MonoidMusician/dhall-purescript)
-- [dhall-rust](https://github.com/Nadrieril/dhall-rust)
-- [dhall-ruby](https://git.sr.ht/~singpolyma/dhall-ruby)
-- [dhall-python](https://github.com/SupraSummus/dhall-python)
+- [dhall-haskell](https://github.com/dhall-lang/dhall-haskell) - Haskell language support
```

Comment:
Hi @Gabriel439 , there are an error with the build.
I think you need to add a full stop at the end of description.
```
  README.md:20:63
  ✖  20:63  List item description must end with proper punctuation  remark-lint:awesome-list-item
  ✖  21:50  List item description must end with proper punctuation  remark-lint:awesome-list-item
  ✖  22:48  List item description must end with proper punctuation  remark-lint:awesome-list-item
  ✖  23:63  List item description must end with proper punctuation  remark-lint:awesome-list-item
  ✖  24:73  List item description must end with proper punctuation  remark-lint:awesome-list-item
  ✖  25:56  List item description must end with proper punctuation  remark-lint:awesome-list-item
  ✖  26:57  List item description must end with proper punctuation  remark-lint:awesome-list-item
  ✖  27:62  List item description must end with proper punctuation  remark-lint:awesome-list-item
```